easybuild/framework/easyblock.py

```diff
@@ -2348,6 +2348,13 @@ def post_install_step(self):
                 run_cmd(cmd, simple=True, log_ok=True, log_all=True)
 
         self.fix_shebang()
+        # GCC linker searches system /lib64 path before the LIBRARY_PATH paths
```

Comment:
`$LIBRARY_PATH` (to make it clear we're referring to the environment variable)

Reply:
Done: https://github.com/easybuilders/easybuild-framework/pull/3401/commits/db3b5e656766b63bcbddfdefe6b0f10ce8cc7ad4